### ISSUE TITLE
.contains() error msg should be more explicit when options are present.

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#49]
When using `cy.contains("...", {visible: true})` and a visible element containing that text was not found, this error message displays:
```
CypressError: Timed out retrying. Could not find any content: 'Loading Dock' in any elements
```

This error would be more accurate if it said "Could not find any content: 'Loading Dock' in any visible elements."

**Example Code**
```coffee
cy.contains("Loading Dock", {visible: true})
```

**Example Console Log**
![screen shot 2015-07-01 at 1 00 24 pm](https://cloud.githubusercontent.com/assets/1271364/8460256/30ee27d0-1ff1-11e5-9bb0-c1b21116f837.png)

**Comment by brian-mann** [comment]
What's actually happening is that it is finding the element containing 'Loading Dock' but it is waiting until it is visible. Because it never becomes visible it then times out. The error message is misleading and additional logic needs to be take taken into account to provide the proper message.

However what you've said: 

> This error would be more accurate if it said "Could not find any content: 'Loading Dock' in any visible elements."

This is inaccurate. The `{visible: true}` command option is not a filter, it is a state that the element must eventually be in. In other words, this is saying: "Once you find the element containing 'Loading Dock' wait until it becomes visible. Throw if it never does.

 If you first wanted to only find Loading Dock in visible elements you would have to do this.

```javascript
cy.get(":visible").contains("Loading Dock")
```

This would first filter and return only currently visible elements, and then find the first element which contains "Loading Dock".

Anyway this update will be released sometime soon.

**Comment by brian-mann** [comment]
Fixed in [`0.9.1`](https://github.com/cypress-io/cypress/wiki/Changelog#091-07032015).
